packages/docs/docs/sdk/Swift/LogtoClient/Enums/LogtoClientErrorTypes.SignIn.md

```diff
@@ -57,4 +57,4 @@ Unable to finish the initial token request after authentication.
 case unexpectedSignInCallback
 ```
 
-The sign in callback URI is not valid.
+The Sign in Callback URI is not valid.
```

Comment:
nit: `in` should also be capitalized. So either `Sign In` or `Sign-in` I think

Reply:
actually it should not be capitalized since OIDC doesn't have this term, reverted